contents/handbook/people/team-structure/website-docs.md

```diff
@@ -23,6 +23,18 @@ We’re here to:
 - Own posthog.com
 - Own the docs
 
+## Q3 2022 Goals
+* **Objective**:
+    * Make posthog.com a great resource for new and returning users
+* **Key Results**:
+    * 25% decrease in support tickets per customer
```

Comment:
This one feels a bit arbitrary, but I'd only consider replacing it with something about growing the volume of organic kudos we get about the site/docs, which also isn't very substantive.

A metric around ticket volume is dependent on three things: 1) Having better docs (which is a near-term goal), 2) Having better community features / more tooling around answer discoverability, 3) Touching the product (eg: building in-app docs search)

I don't mind leaving this one in, though it might take longer than one quarter to achieve.